app.rb

```diff
@@ -34,4 +35,16 @@
   http_status, data = EmailStats.new(params).process
   status http_status
   data
-end
\ No newline at end of file
+end
+
+post '/copy_file' do
```

Comment:
we probably want to namespace this endpoint like `/s3/copy_file` the only reason other name spaces are not present is because the other routes have been single calls